games/o.yaml

```
@@ -1422,6 +1422,28 @@
   updated: 2015-04-15
   url: http://identicalsoftware.com/ostrichriders/
 
+- name: Outpost HD
+  type: remake
```

```suggestion
  type: inspired
```
Looks like it's aiming for a different experience than the original, so this is more appropriate.
From the forum post:
> OutpostHD is a remake of OUTPOST by Sierra On-line from 1994. It's not a clone of OUTPOST -- it's an entirely new game inspired by OUTPOST's core mechanics while providing a more complete and satisfying experience than the original ever could.